recipe/meta.yaml

```diff
@@ -26,11 +26,13 @@ requirements:
     - setuptools >=61
     - wheel >=0.37
     - pybind11 >=2.10.1,<3
-    - qpdf 11
+    - qpdf
   run:
     - python
     - pillow >=10.0.1
     - deprecated
+    # TODO: Remove when `qpdf` version `12` is added.
+    # xref: https://github.com/conda-forge/pikepdf-feedstock/issues/124
     - {{ pin_compatible("qpdf") }}
```

Comment:
Added a note on when this line can later be dropped

xref: https://github.com/conda-forge/pikepdf-feedstock/issues/124